components/builder-api/src/server/mod.rs

```diff
@@ -211,3 +211,9 @@ pub async fn run(config: Config) -> error::Result<()> {
     };
     Ok(srv.run().await?)
 }
```

Comment:
I think there is a better way here, but I don't have that solution off the top of my head now. I'm guessing this is what clippy is complaining about. If you revert the `bitflags` bump, that should eliminate the need for this clone.